Modules/AbRouter/Services/Events/SimpleStatsService.php

```diff
@@ -69,35 +80,50 @@ public function getStats(StatsQueryDTO $statsQueryDTO): StatsResultsDTO
         
         $uniqUsersCount = count($uniqUsers);
         
-        $eventCounters = $this->getCounters(
-            $allUserEvents,
-            $uniqUsers,
-            'event'
+        $eventCounters = $this
+            ->statsFactory
+            ->getStatsMethod('event')
+            ->getCounters(
+                $allUserEvents,
+                $uniqUsers,
+                $allDisplayEvents
         );
 
-        $eventCountersWithDate = $this->getCounters(
-            $allUserEvents,
-            $uniqUsers,
-            'event',
-            true
-        );
+        $eventCountersWithDate = $this
+            ->statsFactory
+            ->getStatsMethod('event')
+            ->getCounters(
+                $allUserEvents,
+                $uniqUsers,
+                $allDisplayEvents,
+                true
+            );
         
-        $eventPercentages = $this->getPercentages(
-            $eventsNames,
-            $eventCounters,
-            $uniqUsersCount
+        $eventPercentages = $this
+            ->statsFactory
+            ->getStatsMethod('event')
+            ->getPercentages(
+                $allDisplayEvents,
+                $eventCounters,
+                $uniqUsersCount
```

Comment:
let's see up, there is a ")" in another position than here. There is a right choice, not here